mock-server/controllers/group.ts

```diff
@@ -0,0 +1,70 @@
+import * as json from '../data/group.json'
+import find from '../utils/find'
+import error from '../utils/error'
+
+/**
+ * '/group/'
+ * @param {object} ctx
+ */
+const index = (ctx) => {
+  const files = ctx.query && ctx.query['files[]']
+  const publicKey = ctx.query && ctx.query.pub_key
+
+  if (!files || files.length === 0) {
+    return error(ctx, {
+      statusText: 'no files[N] parameters found.'
+    })
+  }
+
+  if (files && files.length > 0) {
+    for (let key in files) {
+      if (files.hasOwnProperty(key)) {
+        const file = files[key]
+        const array = file.split('/')
+        const uuid = array[0]
+
+        if (uuid.length < 36) {
+          return error(ctx, {
+            statusText: `this is not valid file url: ${file}`
```

Comment:
I think it's better to write that an id has the wrong format, because it's not clear what the problem is: the wrong format or such an id does not exist or smth else

Reply:
I've just copied this error from API refs : )